File: lib-content-scrapers/src/main/java/com/ustadmobile/lib/contentscrapers/africanbooks/AfricanBooksResponse.java
package com.ustadmobile.lib.contentscrapers.africanbooks;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class AfricanBooksResponse {

    @Expose
    @SerializedName("other")
    public String other;
    @Expose
    @SerializedName("approved")
    public String approved;
    @Expose
    @SerializedName("app")
    public boolean app;
    @Expose
    @SerializedName("dual")
    public boolean dual;
    @Expose
    @SerializedName("level")
    public String level;
    @Expose
    @SerializedName("lang")
    public String lang;
    @Expose
    @SerializedName("people")
    public String people;
    @Expose
    @SerializedName("author")
    public String author;
    @Expose
    @SerializedName("summary")
    public String summary;
    @Expose
    @SerializedName("date")
    public String date;
    @Expose
    @SerializedName("title")
    public String title;
    @Expose
    @SerializedName("id")
    public String id;
}

File: lib-content-scrapers/src/jvmMain/kotlin/com/ustadmobile/lib/staging/contentscrapers/africanbooks/AfricanBooksResponse.kt
package com.ustadmobile.lib.contentscrapers.africanbooks

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class AfricanBooksResponse {

    @Expose
    @SerializedName("other")
    var other: String? = null
    @Expose
    @SerializedName("approved")
    var approved: String? = null
    @Expose
    @SerializedName("app")
    var app: Boolean = false
    @Expose
    @SerializedName("dual")
    var dual: Boolean = false
    @Expose
    @SerializedName("level")
    var level: String? = null
    @Expose
    @SerializedName("lang")
    var lang: String? = null
    @Expose
    @SerializedName("people")
    var people: String? = null
    @Expose
    @SerializedName("author")
    var author: String? = null
    @Expose
    @SerializedName("summary")
    var summary: String? = null
    @Expose
    @SerializedName("date")
    var date: String? = null
    @Expose
    @SerializedName("title")
    var title: String? = null
    @Expose
    @SerializedName("id")
    var id: String? = null
}
